aws/lambda/github-status-webhook-handler/lambda_function.py

```diff
@@ -124,12 +127,12 @@ def lambda_handler(event, context):
         except TypeError:
             print(json.dumps(body))
             committer = "(unknown)"
-        # master commits are always made by facebook-github-bot
-        is_master = committer == "facebook-github-bot"
+        # default branch commits are always made by facebook-github-bot
+        is_default_branch = committer == "facebook-github-bot"
```

Comment:
@suo Our new HUD doesn't use this anymore, correct? If the HUD no longer uses this file, I don't think anything else uses this functionality + we can just delete this lambda.

Reply:
To rephrase the question: does anyone use these stats in the ossci-job-status bucket?

Reply:
don't think so

Reply:
Removed.